docs/Userguide_datasets.rst

```diff
@@ -74,6 +110,41 @@ which requires to first drop the dataset in
    * Multiple archives can be provided to spread the data (e.g. dataset splits,
      raw data, extra data, ...)
 
+Download a dataset from Mila's Google Drive with  ``gdown``
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+The utility gdown is a simple utility to download data from Google Drive. A
+limitation however is that it uses a shared client id which can cause a quota
+block when too many users uses it in the same day. It is described in a `GitHub
+issue <https://github.com/wkentaro/gdown/issues/43#issuecomment-642182100>`_.
+If this is not too big as an issue, a download is very simple, can be done in a
+command line shell or in a Python script and requires no setup.
+
+Download a dataset from Mila's Google Drive with ``rclone``
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+Rclone allows to specify a client id to avoid sharing with other users which
+avoid quota limits. Rclone describes the creation of a `client id in its
+documentaton <https://rclone.org/drive/#making-your-own-client-id>`_. Once this
+is done, a remote for Mila's Google Drive can be configured from the command
+line:
```

Comment:
```suggestion
`Rclone <https://rclone.org/>`_ is a command line program to manage files on
cloud storage. In the context of a Google Drive remote, it allows to specify a
client id to avoid sharing with other users which avoid quota limits. Rclone
describes the creation of a `client id in its documentaton
<https://rclone.org/drive/#making-your-own-client-id>`_. Once this is done, a
remote for Mila's Google Drive can be configured from the command line:
```